Entities/TDExplosive.cpp

```diff
@@ -67,17 +55,10 @@ namespace RTE {
 
 		if (m_Activated) {
 			// If not animated manually, play 'fuse lit' animation
-			if (!m_IsAnimatedManually) {
-				m_SpriteAnimMode = ALWAYSLOOP;
-			} else {
-				// If animated manually, stay on starting frame
-				m_Frame = 0;
-				m_SpriteAnimMode = NOANIM;
-			}
+			m_SpriteAnimMode = !m_IsAnimatedManually ? ALWAYSLOOP : NOANIM;
```

Comment:
I think so.
The only change that *should* happen due to this is that if you stop updating the frame in lua it will stay at the last set frame until you start updating again instead of going to frame 0.

Reply:
Yeah I think that's reasonable behaviour. Cool.